spec/support/config/kaminari.rb

```diff
@@ -0,0 +1,3 @@
+Kaminari.configure do |config|
+  config.default_per_page = 2
+end
```

Comment:
As mentioned earlier, I find depending on global Kaminari configuration problematic. We might be sharing Kaminari with other libraries in the application. If you are unhappy with hard coding, then let's add an option to `Pageflow::Configuration`. That can then also be adapted here for testing purposes without risking reconfiguring some Kaminari usages in unrelated features.